addon/styles/addon.css

```diff
@@ -0,0 +1,145 @@
+.ember-tooltip,
+.ember-popover {
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  user-select: none;
+  pointer-events: none;
+
+  font-size: 14px;
+  font-family: inherit;
+  border-radius: 3px;
+}
+
+.ember-popover {
+  color: #000;
+  background-color: #fff;
+  padding: 10px 20px;
+  border: 1px solid #ccc;
+}
+
+.ember-popover[aria-hidden="false"] {
+  pointer-events: auto;
+  cursor: initial;
+  -webkit-touch-callout: auto;
+  -webkit-user-select: auto;
+  user-select: auto;
+}
+
+.ember-tooltip {
+  max-width: 200px;
+  padding: 6px 10px;
+  color: #fff;
+  background: #3a3c47;
+  text-shadow: -1px -1px 0 rgba(0, 0, 0, 0.2);
+  white-space: normal;
+  white-space: initial;
+}
+
+.ember-tooltip:after {
+  content: '';
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  margin: -5px;
+  background: inherit;
+  transform: rotate(45deg);
```

Comment:
Main thing to be aware of with the removal of the `autoprefixer` dep is we drop support for a couple of [lesser-used old browsers](https://caniuse.com/#feat=transforms2d). To be extra safe, perhaps we can add in `-webkit` prefixes for `transform` and `transition` rules. However, I don't really mind either way since:

1) This is already supporting graceful fallbacks
2) We can deal with this if and when an issue is reported

So I'll leave the addition of prefixes up to @chrism's discretion.

Reply:
Hi @sir-dunxalot, thanks for the review.

My understanding is that `transform` & `transition` only remain unsupported by  [IE8 and opera mobile](http://doiuse.herokuapp.com/?url=&browsers=&css=transform%3A%20rotate(45deg)%3B).

Using a prefix (such as `-webkit`) wouldn't solve this for those two browsers (would require polyfilling I think) and as Ember 2.0 itself no longer supports IE8 I'd hoped it wouldn't be necessary to support it in this addon.

Also, another consideration is that as the `transform` is located in a pseudo element selector (`:after`) I don't think IE8 supports that anyway.

If I've got some of this wrong I'd be happy to update, but I don't think there is much possible for the reasons I've listed.

Thanks.